cmd/copilot/template/template.go

```diff
@@ -29,6 +29,11 @@ var RootUsage = fmt.Sprintf("{{h1 \"Commands\"}}{{ $cmds := .Commands }}{{$group
 {{.InheritedFlags.FlagUsages | trimTrailingWhitespaces}}{{end}}{{if .HasExample}}
 
 {{h1 "Examples"}}{{code .Example}}{{end}}
+
+{{h1 "Get Involved"}}
+  Be a part of the Copilot ✨community✨! 
```

Comment:
How are the emojis rendered in the terminal? Can we see what the output looks like?